source_files/gpu.c

```diff
@@ -79,27 +98,55 @@ static int bbs(void) {
     return s;
 }
 
-static void* startBurn(void *index) {
+void* fillup(void* array, int useD, int size) {
```

Comment:
make static, do not pass array, but return it

Reply:
why does this version of the function revert fixes already in master? e.g. check for malloc?